.github/workflows/local-dev-ci.yaml

```
@@ -0,0 +1,58 @@
+name: Execute local test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "docs/**"
+      - "**/*.md"
+
+jobs:
+  local-dev-test:
+    name: local-dev-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: v1.17
+
+      # Install kind with a local registry
+      - uses: container-tools/kind-action@v1
+        name: Kubernetes KinD Cluster w/local registry
```

```suggestion
        name: Kubernetes kind Cluster w/local registry
```
Nothing much important, but if there is no specific meaning of `KinD` then please change.

---

fxied.